connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConfigTransformerTest.java

```diff
@@ -69,18 +72,18 @@ public void testReplaceVariable() {
     @Test
     public void testReplaceVariableWithTTL() {
         EasyMock.expect(worker.herder()).andReturn(herder);
-        EasyMock.expect(herder.connectorConfigReloadAction(MY_CONNECTOR)).andReturn(Herder.ConfigReloadAction.NONE);
 
         replayAll();
 
-        Map<String, String> result = configTransformer.transform(MY_CONNECTOR, Collections.singletonMap(MY_KEY, "${test:testPath:testKeyWithTTL}"));
-        assertEquals(TEST_RESULT_WITH_TTL, result.get(MY_KEY));
+        Map<String, String> props = new HashMap<>();
```

Comment:
would it better to explicitly check that herder.connectorConfigReloadAction(connectorName) is never invoked as part of the tests and include a comment on why it shouldn't be invoked. This would provide some kind of safety net when someone tries to accidentally put it back.

Reply:
Good idea, I've added checks to the tests to ensure that connectorConfigReloadAction is not called.